include/cutlass/gemm/collective/builders/pvc_dpas_builder.inl

```diff
@@ -0,0 +1,128 @@
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * 3. Neither the name of the copyright holder nor the names of its
+ * contributors may be used to endorse or promote products derived from
+ * this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ **************************************************************************************************/
+
+#pragma once
+
+#include <cutlass/arch/arch.h>
+#include <cutlass/gemm/dispatch_policy.hpp>
+
+#include "cutlass/gemm/collective/collective_mma.hpp"
+
+namespace cutlass::gemm::collective {
+
+  // Intel PVC Single stage dpas pipeline
+  // Also the auto builder
+
+template <
+  class ElementA,
+  class GmemLayoutATag,
+  int AlignmentA,
+  class ElementB,
+  class GmemLayoutBTag,
+  int AlignmentB,
+  class ElementAccumulator,
+  class TileShape_MNK,
+  class KernelScheduleType
+  > 
+struct CollectiveBuilder<
+  arch::IntelPVC,
+  arch::OpClassTensorOp,   // Reusing opClassTensorOp instead of declaring a new "OpClassDpasOP" as they have the same meaning
+  ElementA,
+  GmemLayoutATag,
+  AlignmentA,
+  ElementB,
+  GmemLayoutBTag,
+  AlignmentB,
+  ElementAccumulator,
+  TileShape_MNK,
+  Shape<_1, _1, _1>,    // Cluster Shape, which has no meaning in PVC
+  cutlass::gemm::collective::StageCountAuto, // Stage Count needn't be a parameter
+  KernelScheduleType,
+  cute::enable_if_t<
+    (cute::is_same_v<KernelScheduleType, KernelSinglestage> ||
+     cute::is_same_v<KernelScheduleType, KernelScheduleAuto>) &&  
+    cute::is_same_v<GmemLayoutATag, cutlass::layout::RowMajor> && // Different struct specialization because this will change copy atoms
+    cute::is_same_v<GmemLayoutBTag, cutlass::layout::RowMajor>
+  >
+    >{
+      #ifndef SYCL_INTEL_TARGET
+        static_assert(cutlass::detail::dependent_false<arch::IntelPVC>, 
```

Comment:
I am not sure if we need this `static_assert` here or not since we have already partially specialized the `CollectiveBuilder` struct with `arch::IntelPVC`.

Reply:
This is for  catching the case when someone uses this PVC struct, even when the target is not intel_gpu_pvc

Reply:
This has now been changed to check if target is Nvidia rather than particularly checking for PVC